docs/badge.html

```diff
@@ -0,0 +1,99 @@
+    
+        </ul>   
+    </aside>
+
+    <div class="main main-container">
+        <div class="description">
+            <p1>Badges are used to highlight an item's status for quick recognition.Badges are usually seen in social media apps to indicate the status of whether a person is online or offline. They are also used to indicate number of unread notifications. 
+            </p1>
+        </div>
+
+
+        <div class="badge-container-size">
+            <p>Badge on icon</p>    
+            <div class="badge-icon">
+                <div><img src="/Images/envelop icon.png"><p2 class="badge badge-icon-one">4</p2></div>
+                <div><img src="/Images/bell icon.png"><p2 class="badge badge-icon-two">2</p2></div>
+                <div><img src="/Images/shopping-cart.png"><p2 class="badge badge-icon-three">5</p2></div>
+            </div>
+
+            <pre>
+                <code class="language-html">
+                    &lt;div class="badge-icon"&gt;
+                        &lt;div&lt;img src="Image source"&gt;&lt;p2 class="badge badge-icon-one"&gt4&lt;/p2&gt;&lt;/div&gt;
+                        &lt;div&lt;img src="Image source"&gt;&lt;p2 class="badge badge-icon-two"&gt2&lt;/p2&gt;&lt;/div&gt;
+                        &lt;div&lt;img src="Image source"&gt;&lt;p2 class="badge badge-icon-three"&gt5&lt;/p2&gt;&lt;/div&gt;
+                     &lt;/div&gt;
+                </code>
+            </pre>
+        </div>
+
+        <div>
+            <p>Badge on avatar</p>
+            <div class="badge-avatar">
+                <div><img src="/Images/1.jpg"><p3 class="badge badge-avatar-one"></p3></div>
+                <div><img src="/Images/2.jpg"><p3 class="badge badge-avatar-two"></p3></div>
+                <div><img src="/Images/1.jpg"><p3 class="badge badge-avatar-three">4</p3></div> 
+                <div><img src="/Images/2.jpg"><p3 class="badge badge-avatar-four">1</p3></div> 
+            </div>
+
+            <pre>
+                <code class="language-html">
+                    &lt;div class="badge-icon"&gt;
+                        &lt;div&lt;img src="Image source"&gt;&lt;p2 class="badge badge-avatar-one"&gt&lt;/p2&gt;&lt;/div&gt;
+                        &lt;div&lt;img src="Image source"&gt;&lt;p2 class="badge badge-avatar-two"&gt&lt;/p2&gt;&lt;/div&gt;
+                        &lt;div&lt;img src="Image source"&gt;&lt;p2 class="badge badge-avatar-three"&gt4&lt;/p2&gt;&lt;/div&gt;
+                        &lt;div&lt;img src="Image source"&gt;&lt;p2 class="badge badge-avatar-four"&gt1&lt;/p2&gt;&lt;/div&gt;
+                     &lt;/div&gt;
+                </code>
+            </pre>
+        </div>
+    </div>
+
+    <div class="footer">
+        <div class="footer-text">designed by Parul Gupta</div> 
+        <div class="social-media">
+            <a href="https://www.linkedin.com/in/parul-gupta-4a1413131/"><img class="social-media-icon" src="/Images/linkedin icon.png"></a> 
```

Comment:
Suggestion: for Footer Social links, you can use icons instead of images as Icons require less styling when compared to icons.